lib/elixir/src/elixir_errors.erl

```diff
@@ -395,6 +398,15 @@ parse_erl_term(Term) ->
   {ok, Parsed} = erl_parse:parse_term(Tokens ++ [{dot, 1}]),
   Parsed.
 
+raise_mismatched_delimiter(Location, File, Input, Message) ->
+  {line, StartLine} = lists:keyfind(line, 1, Location),
+  {column, StartCol} = lists:keyfind(column, 1, Location),
+  {end_line, EndLine} = lists:keyfind(end_line, 1, Location),
+  {end_column, EndCol} = lists:keyfind(end_column, 1, Location),
+  {InputString, InputStartLine, _} = Input,
+  Snippet = snippet_line(InputString, [{line, EndLine}, {column, EndCol}], InputStartLine),
+  raise('Elixir.MismatchedDelimiterError', Message, [{file, File}, {snippet, Snippet}, {start_position, {StartLine, StartCol}}, {end_position, {EndLine, EndCol}}]).
```

Comment:
Sorry for the nitpick, but can we keep the fields line, column, end_line, and end_column? This is mostly to mirror other exceptions that use line and column as well, making them more consistent. Unless we don't want those to match on purpose and have someone use only the line/column by accident? But I don't think nothing would be wrong with that.